integr8ly/integreatly-operator
# feat: Creates s3 credentials and bucket automatically

Automates the creation of s3 credentials and bucket 

https://issues.jboss.org/browse/INTLY-4009

## Verification steps

1. Run a full install of the integreatly-operator:
`operator-sdk up local --namespace=<integreatly_operator_namespace>`

2. Verify if the `s3-credentials` and `s3-bucket` secrets are created:
`oc get secret`

3. Ensure installation of all products has been completed successfully
